### PR TITLE
integartion tests for get and post requests of categories under admin…

### PR DESCRIPTION
**Pull request by harshithaputtaswamy**
initial code structure and tests for get and post requests of categories under admin apis

### CODE REVIEW

File: test/common/mock_admin_data.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (Shwetabhk) [inline test/common/mock_admin_data.json:1]: Let's have a different folder under `test` and move all the files related to each of these APIs there
```
admin/
spender/
approver/
common/
```

File: test/integration/test_fyle_admin.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  }
+
+  accounting_exports_generator = fyle.v1beta.admin.accounting_exports.list_all(query_params=query_params)
+  mock_accounting_exports = mock_data.accounting_export.get()
+
+  for response in accounting_exports_generator:
+    if response.get('data'):
+      accounting_export.extend(response['data'])
+
+  assert dict_compare_keys(accounting_export[0], mock_accounting_exports[0]) == [], 'fyle.v1beta.admin.accounting_exports.list_all() has stuff that mock_data doesnt'
```
Comment (ashwin1111) [inline test/integration/test_fyle_admin.py:27]: handle null cases as well, do all these checks only when you have at least 1 `accounting_export`

File: test/integration/test_fyle_admin.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+  categories_generator = fyle.v1beta.admin.categories.list_all(query_params=query_params)
+  mock_categories = mock_data.categories.get()
+
+  for response in categories_generator:
+    if response.get('data'):
+      categories.extend(response['data'])
+
+
+  assert dict_compare_keys(categories[0], mock_categories[0]) == [], 'fyle.v1beta.admin.categories.list_all() has stuff that mock_data doesnt'
```
Comment (ashwin1111) [inline test/integration/test_fyle_admin.py:52]: same comment as before

Comment (ashwin1111) [inline test/integration/test_fyle_admin.py:52]: same comment as before

File: test/integration/test_fyle_admin.py [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      "is_enabled": True,
+      "system_category": "Others",
+      "code": "C1234",
+      "restricted_project_ids": [
+        247690, 247593
+      ]
+    }
+  })
+  mock_categories = mock_data.categories.get()
+  print(add_category)
```
Comment (ashwin1111) [inline test/integration/test_fyle_admin.py:70]: once the PR is ready to be finalised, remove the loggers

Comment (ashwin1111) [inline test/integration/test_fyle_admin.py:70]: once the PR is ready to be finalised, remove the loggers

**Comment by ashwin1111** [comment]
taken care in https://github.com/fylein/fyle-platform-sdk-py/pull/43

**Pull request closed without merging**